podium-gateway/src/main/java/nl/thehyve/podium/domain/ReviewRound.java

```diff
@@ -85,10 +85,20 @@
 
     @CreatedDate
     @Column(name = "start_date", nullable = false)
-    private ZonedDateTime startDate = ZonedDateTime.now();
+    // FIXME:
+    // ZonedDateTime is no longer supported;
+    // LocalDateTime is not supported yet;
```

Comment:
What do you mean with 'not supported yet'?

Reply:
It means that older Spring Data / Spring JPA did not support `java.time.LocalDateTime` for fields with auditing annotations like `@CreatedDate`. (`ZonedDateTime` compatibility was dropped in Spring Boot 2)
But with SB 2.2 I can use `LocalDateTime`, so this is fixed now.